_drafts/2016-12-21-commcallv12-review-coc.md

```diff
@@ -0,0 +1,84 @@
+* Anonymous reporting of a violation should be an option.
+* Progression of consequences can move from a verbal warning, to a time-based ban, to a permanent ban, all the way to legal action with real-world authorities.
+
+In the realm of CoCs, enforcement is the hard part, and the part that is most often overlooked until a crisis forces the issue. Enforcement should be handled by a board of three to five people and have representation of different perspectives. For an event, that board could include the event organizer, a community member, and someone who was involved in developing the CoC. The people involved in enforcement must have empathy for the issue being reported so it's important to involve people who don't necessarily have power within the community or organization. Someone suggested that an enforcement board might be wise to include someone from outside their own community.
+
+In an academic research environment, there is a huge grey area between dealing with a violation within a lab and getting the university involved. This grey area can be filled by identifying a person outside the lab AND outside the university to participate when needed. This third party can be the first point of contact for complaints about the principal investigator, for example. A lot can be resolved before escalating to involvement of the university, its lawyers, or unions. (Of course there are cases where this is warranted.)
+
+One tricky area of enforcement is around online conversations in discussion forums or in GitHub issues where people sometimes say things they shouldn’t. Safia made the excellent point that it helps to leave a comment on the issue because it will remain visible online. Saying something like “we're willing to help, but we don’t tolerate this kind of communication” means newcomers can see that bad behavior is addressed and not tolerated. This is also critical to maintain or rebuild trust from community members. If you're a GitHub admin you could choose to implement a time-based ban as a consequence. In this situation, recognition of a violation is easy, but there is a big enforcement challenge when that person is a significant contributor. This is where it is critical to have a strong enforcement board and a good structure for raising a violation through the ranks depending on its severity.
+
+## We Have Work To Do
+rOpenSci currently has a [Code of Conduct for our events](http://unconf16.ropensci.org/coc.html) and for [software onboarding](https://github.com/ropensci/onboarding/blob/master/policies.md/#code-of-conduct). We'll take what we learned in this Community Call to develop a more comprehensive CoC, make sure it's easily findable and that consequences of violations are enforceable. [Let us know](https://ropensci.org/contact.html) if you'd like to get involved.
```

Comment:
thanks for mentioning that one - good to show! 
I was thinking of it and looking but couldn't find cause didn't remember it applied to individual packages! Didn't remember enough to ask about it. When I re-found the software onboarding CoC, thought that was it